content/uiux/concepts/research-plan/research-plan.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: 'Research Plan'
+Description: 'A research plan is a deliverable that outlines the goals, methods, and key details of a research study.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A **research plan** is a deliverable that outlines the goals, methods, and key details of a research study. Writing a research plan is generally the first step of conducting [user research](https://www.codecademy.com/resources/docs/uiux/user-research). Documenting this plan helps teams align on essential details and define a strategic framework.
+
+## Sections
+
+A research plan should cover:
```

Comment:
Remove blank lines between elements in the bulleted list.

Reply:
Removed